app/cdap/components/Administration/TetheringTabContent/OdfTetheringConnections/index.tsx

```diff
@@ -42,13 +45,40 @@ const OdfTetheringConnections = (): JSX.Element => {
     // TODO: navigate to create request page
   };
 
+  const [connections, setConnections] = useState([]);
+  const [pendingRequests, setPendingRequests] = useState([]);
+
+  const fetchConnectionsList = async () => {
+    try {
+      const list = await TetheringApi.getTetheringStatusForAll().toPromise();
+      const establishedConnections = [];
+      const pendingConnections = [];
+
+      list.map((conn: IConnection) =>
```

Comment:
Prefer `forEach` if you're not actually returning a 1:1 map (or lodash `groupBy` for this use case).